tools/policy-gen/generator/pkg/parse/policyconfig.go

```diff
@@ -119,7 +119,7 @@ func newPolicyConfig(pkg, name string, markers map[string]string, hasTo, hasFrom
 		Name:                name,
 		NameLower:           strings.ToLower(name),
 		SingularDisplayName: core_model.DisplayName(name),
-		PluralDisplayName:   core_model.PluralDisplayName(name),
+		PluralDisplayName:   core_model.PluralType(core_model.DisplayName(name)),
```

Comment:
should we remove `PluralDisplayName`?

Reply:
Ah oops, yeah should be one or the other here of course. I'll get rid of it